src/decorators/authorizedUser.ts

```diff
@@ -0,0 +1,8 @@
+import { ExecutionContext, createParamDecorator } from '@nestjs/common';
+
+import { UserInfoDto } from '../user/dto/user.dto';
+
+export const AuthorizedUser = createParamDecorator((data, context: ExecutionContext): UserInfoDto => {
+  const req = context.switchToHttp().getRequest();
+  return req.user;
+});
```

Comment:
Here it says it returns `UserInfoDto` but in the controller, the type is set to `TokensDto`. Why?